### PR TITLE
Add screenshot to README.md

### PR DESCRIPTION
**Pull request by Damillora**
Made as part of Google Code In 2017

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 Torrentor!
 =============
 
+![Torrentor app](torrentor.png "Torrentor")
```
Comment (humdingerb) [inline README.md]: The screenshot would look beter after the introductory sentence.

Comment (humdingerb) [inline README.md]: While you're updating the readme, a few things to improve:

"To build Torrentor you need some libs which are optional packages."
-> "To build Torrentor you need some development libraries:"

* OpenSSL   -- ```$ pkgman install devel:libssl```
* libevent  -- ```$ pkgman install devel:libevent```
* curl      -- ```$ pkgman install devel:libcurl```

Also add a linebreak in the next to last line a set full-stops.

Comment (Damillora) [reply README.md]: Changed it.

Comment (Damillora) [reply README.md]: Updated

Comment (humdingerb) [reply README.md]: You forgot to change the "lib:" parts of the pkgman lines to "devel:". To build the application we need the dev libs, not the runtime libs.

Comment (Damillora) [reply README.md]: Ah, updated.

**Comment by humdingerb** [comment]
Thanks.

**Pull request merged**